src/test/java/com/redhat/cloud/custompolicies/app/RestApiTest.java

```diff
@@ -307,6 +314,7 @@ void storeNewPolicy() {
 
     assert body.get("conditions").equals("cores = 2");
     assert body.get("name").equals("test1");
+    assert (Integer)body.get("id") == returnedBody.id;
```

Comment:
Hopefully this isn't autoincrement integer?

Reply:
It may be under the covers, but JPA is taking care of it. 
If this int would only be incremented after the store being completed, this assert would fail.
Also GETting the policy by id (location) would fail before.

Reply:
That was not the point, but the fact that it reveals potentially too much. For example if the service is not used by many customers, it can be easily seen from the new id one generates. I'm not sure if that's wanted behavior.

Reply:
We can change that - can you please open a JIRA for it? The change should not be part of this PR (imo), as this is about something else.

Reply:
I have opened CPOL-92 for the Id change to UUID.